script/Arcueid/Day 8/08_00_ARC08_2.txt

```diff
@@ -216,9 +220,10 @@ Although we both have something to gain by working together, I find it hard to s
 [sha:8ad97f108ab8b0847cea33d1d04781331c3901a4]{
 -- Page 71, Offset 19851.
 -- 「でしょうね。今まで見た事もないほどの慌てようでしたから。何事も真剣になってくれない兄さんが感情を露にするなんて、どういう風の吹き回しかと疑ったほどです」
-"Of course. After all, I've never seen you in a panic like that before.
- For my otherwise serious brother to reveal his emotions,
- naturally I would wonder what on earth could have brought that on."
+// 風の吹き回し I assume turn of events rather than her stroke of luck at seeing him flustered?
+"I thought as much.
+ After all, I've never seen you in a panic like that before.
+ Seeing my ordinarily emotionless brother so flustered was enough to make me suspicious."
```

Comment:
どういう風の吹き回しか is a set-phrase
https://www.weblio.jp/content/%E3%81%A9%E3%81%86%E3%81%84%E3%81%86%E9%A2%A8%E3%81%AE%E5%90%B9%E3%81%8D%E5%9B%9E%E3%81%97%E3%81%8B

Reply:
Right, but the idea is like "oh, that's rare", right? I thought the 疑ったほど was like its rare -> suspiciously rare

Reply:
I mean yeah,
this comment was mostly because it seemed like the comment you left was unclear about what 風の吹き回し was meant to signify and I wanted to clarify. I don't particularly object to the TL